### PR TITLE
Admin login field Validator

### PR DESCRIPTION
**Pull request by Se7en-et**

### CODE REVIEW

File: src/pages/admin/Login.vue [hunk 1]
```diff
@@ -8,12 +8,14 @@
...
-      <q-input v-model="password" float-label="Password" color="red-9" type="password" />
+      <q-input clearable v-model="username" float-label="Username" color="secondary" :error="$v.username.$error" />
+      <et-validator :dirty="$v.username.$dirty" :show="!$v.username.required" msg="Username is required" />
+      <q-input v-model="password" float-label="Password" color="secondary" type="password" :error="$v.password.$error" />
+      <et-validator :dirty="$v.password.$dirty" :show="!$v.password.required" msg="Password is required" />
     </q-card-main>
     <q-card-actions>
-      <div class="row justify-center btn-row">
-        <q-btn :loading="getIsLoading" color="amber-3" label="Sign In" class="text-brown-6 q-ma-sm col-10" @click="loginAdmin({username,password})">
+      <div class="row justify-center" style="height:60px;width:100%;">
```
Comment (hemvanh) [inline src/pages/admin/Login.vue]: pls create a stylus class for this :  height:60px;width:100%;

**Review by hemvanh — CHANGES_REQUESTED** [review]

**Comment by phong-et** [comment]
I changed

**Review by hemvanh — CHANGES_REQUESTED** [review]
pls help update changelog also

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file.
 - Admin Default layout & Login
 - Customer Detail Page with Sign-Out feaature
 - Schema Admin (res loginAdmin,def type Admin,input AdminLoginInput)[@john]
+- Admin validator
```
Comment (hemvanh) [inline CHANGELOG.md]: Admin validator  -> Admin Login validator

**Review by hemvanh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Se7en-et) [reply CHANGELOG.md]: ok a

**Pull request merged**